simplicity-sys/src/c_jets/c_env.rs

```diff
@@ -199,6 +201,12 @@ extern "C" {
     pub fn c_free_transaction(tx: *mut CTransaction);
     pub fn c_free_tapEnv(env: *mut CTapEnv);
 }
+impl CElementsTxEnv {
+    pub fn sighash_all(&self) -> sha256::Hash {
```

Comment:
nit: Not to be done in this PR. But ideally, we would want this to new `hash_newtype` like `SimplicitySigHash`